docs/concepts/README.md

```diff
@@ -7,9 +7,10 @@ abstractions that Kueue uses to represent your cluster and workloads.
 
 ### [Resource Flavor](resource_flavor.md)
 
-A kind or type of resource in a cluster. It could distinguish among different
-characteristics of resources such as availability, pricing, architecture,
-models, etc.
+A kind or type of resource that you can define in a cluster. Typically,
```

Comment:
`that is available in a cluster`

Or `that you can define to describe what is available in the cluster`?

Reply:
Changed the wording a bit, following also the definition under resourceflavor.md